invenio_records_rest/serializers/loaders/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this should go to ``invenio_records_rest/loaders/__init__.py`` similar to how it was in the data model package.